flink-dist/pom.xml

```diff
@@ -305,6 +305,7 @@ under the License.
 				<!--Build uber jar-->
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-shade-plugin</artifactId>
+				<version>2.4.1</version>
```

Comment:
Are you sure the wrong version was used here?
I intentionally removed the version tag of the `maven-shade-plugin` from the sub-modules and set the version centrally in the `dependencyManagement` section of the main pom.